scripts/plot-package-degrees.py

```diff
@@ -0,0 +1,26 @@
+"""
+Fetches all dependencies (first order) of each package and 
+plots the histogram of the number of dependencies (degree)
+
+
+"""
+import glob
+import pylab
+import json
+
+
+
+degrees = []
+for filename in glob.glob("package-data/*json"):
```

Comment:
Please put the degree information into the input file ("package-data/all.tsv") as column "deps".

Reply:
Then, you can use it here directly instead of parsing the json files again.